features/issues/components/issue-filter/issue-filter.tsx

```diff
@@ -0,0 +1,161 @@
+import {
+  Button,
+  ButtonSize,
+  ButtonColor,
+  ButtonVariant,
+  ButtonIcon,
+  SelectBox,
+  InputBox,
+  InputIcon,
+} from "@features/ui";
+import { IssueLevel, IssueStatus } from "@api/issues.types";
+import capitalize from "lodash/capitalize";
+import styles from "./issue-filter.module.scss";
+import { useRouter } from "next/router";
+import { useGetIssues } from "@features/issues";
+import { useRef, useEffect, useState } from "react";
+
+export function IssueFilter({ showButton = true }) {
+  const issueLevels = Object.values(IssueLevel).map((level) => ({
+    value: level,
+    label: capitalize(level),
+  }));
+
+  const issueStatus = Object.values(IssueStatus).map((status) => ({
+    value: status,
+    label: capitalize(status),
+  }));
```

Comment:
You could move these outside the component. That way the resulting objects would be static and wouldn't cause a re-render somewhere down the line. I mean like this

```
const issueLevels = Object.values(IssueLevel).map((level) => ({
  value: level,
  label: capitalize(level),
}));

const issueStatus = Object.values(IssueStatus).map((status) => ({
  value: status,
  label: capitalize(status),
}));

export function IssueFilter({ showButton = true }) {
  ...
```